src/main/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoader.java

```diff
@@ -60,6 +62,14 @@ protected void init() {
                     activeVenueDetailsMapByEpimsId.put(line[15], venueDetails);
                 }
             });
+            activeVenueEpimsIdsMapByRpc.putAll(
+                    activeVenueDetailsMapByEpimsId // use active otherwise closed epimsIds (000000) picked up
```

Comment:
Minor, your using 8 indent instead of 4, you might want to change your intellij settings

Reply:
We'll let it slide.